Source/PLCrashReporter.h

```diff
@@ -122,6 +122,7 @@ typedef struct PLCrashReporterCallbacks {
 + (PLCrashReporter *) sharedReporter PLCR_DEPRECATED;
 
 - (instancetype) initWithConfiguration: (PLCrashReporterConfig *) config;
+- (instancetype) initWithConfiguration: (PLCrashReporterConfig *) config basePath:(NSString*) basePath;
```

Comment:
Formatting nit - extra space
```suggestion
- (instancetype) initWithConfiguration: (PLCrashReporterConfig *) config basePath:(NSString *) basePath;
```